src/payment/checkout/Checkout.test.jsx

```diff
@@ -106,19 +109,24 @@ describe('<Checkout />', () => {
 
     // Apple Pay temporarily disabled per REV-927 - https://github.com/openedx/frontend-app-payment/pull/256
 
-    // TODO: Disabling for now update once we can swap between stripe and cybersource
-    // it('submits and tracks the payment form', () => {
-    //   const formSubmitButton = wrapper.find('form button[type="submit"]').hostNodes();
-    //   formSubmitButton.simulate('click');
-
-    //   expect(sendTrackEvent).toHaveBeenCalledWith('edx.bi.ecommerce.basket.payment_selected', {
-    //     type: 'click',
-    //     category: 'checkout',
-    //     paymentMethod: 'Credit Card',
-    //     checkoutType: 'client_side',
-    //     flexMicroformEnabled: true,
-    //   });
-    // });
+    it('submits and tracks the payment form', () => {
+      expect(sendTrackEvent).toHaveBeenCalledWith('edx.bi.ecommerce.payment_mfe.payment_form_rendered', {
+        ...getPerformanceProperties(),
+        paymentProcessor: 'Cybersource',
```

Comment:
Is see that the payment processor is Cybersource, I am surprised this is passing considering there is nothing about the flag to determine if it should render which payment form?

Reply:
the cybersource payment form is shown by default if there is no flag.  All the cybersource elements check for is not loading and not stripe enabled Checkout.jsx lines 220 and 280.

Reply:
Gotcha, that's true, when the flag is `false` and not loading it will render the CyberSource, great thanks!